CONTRIBUTING.md

```diff
@@ -93,9 +93,6 @@ To overwrite them, use `compileJavaJvmArgs` Gradle project property
 (e.g. `./gradlew -PcompileJavaJvmArgs='-Xmx2g -XX:+HeapDumpOnOutOfMemoryError' build`,
 or equivalently with an env var: `ORG_GRADLE_PROJECT_compileJavaJvmArgs='-Xmx2g -XX:+HeapDumpOnOutOfMemoryError' ./gradlew build`).
 
-By default, Lombok's annotation processor runs on the fly and Delomboked sources are **not** saved to <subproject>/build/delombok/...<br/>
-To enable Delombok, set `useDelombok` Gradle project property (e.g. `./gradlew -PuseDelombok build`).
```

Comment:
That didn't even work anymore, so probably nobody used that in a long time :trollface: